Main.java

```diff
@@ -0,0 +1,66 @@
+package com.company;
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        if (args.length != 2) {
+            System.out.println("File path and search word cannot be empty");
+            return;
+        }
+        String textFilePath = args[0];
+        String searchKeyword = args[1];
+        File file = new File(textFilePath);
+        System.out.println("Processing...");
+        validateFile(file);
+        if (validateFile(file)) {
+            wordCount(file, searchKeyword);
+        }
+    }
+
+    private static boolean validateFile(File file) {
+
+        if (!file.exists()) {
+            System.out.println("File does not exist");
+
+        } else if (!(file.getName().endsWith(".txt") || file.getName().endsWith(".json"))) {
+            System.out.println("File format not supported");
+            System.exit(0);
+        }
+        return true;

```

Comment:
Why is a true being returned here? It does not seem to have anything to do with whether the file is valid or not

Reply:
I have changed the method to return true when the file type is either a .txt or .json file and false when it is of any other type.